docs: connected local UI (#3764)
spectrocloud/librarium: docs/docs-content/clusters/edge/local-ui/cluster-management/export-cluster-definition.md

## Export Cluster Definition with Palette API

> If you don't have a AMC64 machine, or you do not want to download and use the Palette Edge CLI, you can still export

Reviewer: ```suggestion
If you do not have an AMC64 machine, or you do not want to download and use the Palette Edge CLI, you can still export
```

Reviewer 2: @lennessyy  Did you mean AMD64?
